galaxy.yml

```diff
@@ -8,7 +8,8 @@ authors:
   - willthames (https://github.com/willthames)
   - mmazur (https://github.com/mmazur)
   - jamescassell (https://github.com/jamescassell)
-dependencies: {}
+dependencies:
+  cloud.common: '*'
```

Comment:
Interesting—it seems like maybe `ansible-test sanity` is not automatically installing this dependency?

Reply:
Indeed, the `ERROR: plugins/modules/k8s_info.py:145:0: traceback: ModuleNotFoundError: No module named 'ansible_collections.cloud'` error is suspicious.